### ISSUE TITLE
Adenomyosis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7583]
**Preferred term label:**
Adenomyosis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Ectopic endometrial tissue within the uterine myometrium.
PMID: 30969690 

**Parent term (use hpo.jax.org/app)**
Abnormal uterus morphology HP:0031105

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 600458 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
